package.json

```diff
@@ -2,10 +2,20 @@
   "name": "cross-domain-utils",
   "version": "1.0.9",
   "description": "Javascript module template.",
-  "main": "index.js",
+  "main": "dist/cross-domain-utils.js",
   "scripts": {
-    "test": "gulp test",
-    "postinstall": "node postinstall.js"
+    "postinstall": "flow-typed install",
+    "build": "babel-node ./node_modules/.bin/webpack --progress --output-path dist",
+    "karma": "babel-node ./node_modules/.bin/karma start",
```

Comment:
its run on the lint task.  Figured they kinda go hand in hand.  Do you still want a separate task?